app/src/main/java/com/amaze/filemanager/ui/dialogs/EncryptAuthenticateDialog.kt

```diff
@@ -95,10 +97,13 @@ object EncryptAuthenticateDialog {
             val useAzeEncrypt: AppCompatCheckBox = vb.checkboxUseAze
             val usageTextInfo: AppCompatTextView =
                 vb.textViewCryptInfo.apply {
+                    movementMethod = LinkMovementMethod.getInstance()
+                    autoLinkMask = Linkify.WEB_URLS
+                    linksClickable = true
                     text =
                         HtmlCompat.fromHtml(
                             main.getString(R.string.encrypt_option_use_aescrypt_desc),
-                            FROM_HTML_MODE_COMPACT,
+                            HtmlCompat.FROM_HTML_MODE_LEGACY,
```

Comment:
why switch back from the recommended `FROM_HTML_MODE_COMPACT` flag to `FROM_HTML_MODE_LEGACY`? ref: https://developer.android.com/reference/android/text/Html.html#FROM_HTML_MODE_COMPACT